server/webhook_parser.go

```diff
@@ -340,7 +340,7 @@ func parseWebhookCommentUpdated(jwh *JiraWebhook) (Webhook, error) {
 		text:        truncate(jwh.Comment.Body, 3000),
 	}
 
-	appendCommentNotifications(wh)
+	appendCommentNotifications(wh, "mentioned you in an update on")
```

Comment:
I feel that this may not get the point across that the `comment` is what was updated. It sounds like the issue was updated, which could have been done with a new comment.

```suggestion
	appendCommentNotifications(wh, "mentioned you in a comment update on")
```
This suggestion doesn't sound great, there's probably a better wording.